### PR TITLE
Seccomp path fixes

### PR DESCRIPTION
**Pull request by runcom**
@thaJeztah @justincormack PTAL

### CODE REVIEW

File: cli/command/system/info.go [hunk 1]
```diff
@@ -147,7 +147,7 @@ func prettyPrintInfo(dockerCli *command.DockerCli, info types.Info) error {
 				case "Name":
 					fmt.Fprintf(dockerCli.Out(), " %s\n", o.Value)
 				case "Profile":
-					if o.Key != "default" {
+					if o.Value != "default" {
 						fmt.Fprintf(dockerCli.Err(), "  WARNING: You're not using the Docker's default seccomp profile\n")
```
Comment (thaJeztah) [inline cli/command/system/info.go]: should probably be changed to either;

```
WARNING: You're not using Docker's default seccomp profile
```

or

```
WARNING: You're not using the default seccomp profile
```

**Review by thaJeztah — COMMENTED** [review]
One nit, but LGTM otherwise

**Comment by thaJeztah** [comment]
Oh, can you squash some commits (especially the last two)?

**Comment by runcom** [comment]
last two are honestly unrelated, but there you go, I squashed 1st and 3rd..

**Comment by justincormack** [comment]
LGTM

**Review by thaJeztah — APPROVED** [review]
> last two are honestly unrelated, but there you go, I squashed 1st and 3rd..

oh, that's what I meant :smile:

LGTM

**Comment by runcom** [comment]
rebased and pushed again, built was green though so feel free to merge this :angel:

**Review by justincormack — APPROVED** [review]

**Pull request merged**